.github/workflows/integration_test.yaml

```diff
@@ -16,14 +16,13 @@ jobs:
     secrets: inherit
     with:
       juju-channel: 3.1/stable
-      pre-run-script: scripts/pre-integration-test.sh
+      pre-run-script: scripts/setup-lxd.sh
```

Comment:
I guess this is setting up LXD to deploy the GitHub runner charm on which then connects to OpenStack for the actual tests

Reply:
Yes, it bootstraps lxd, nothing related to the charm.